LondonBritishLibrary/orient/BLorient757.xml

```diff
@@ -31,23 +31,126 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         <altIdentifier>
                             <idno>Wright 335</idno>
-                            
+
                         </altIdentifier>
-                        
+
                     </msIdentifier>
                     <msContents>
                         <summary/>
-                        
-                        
-                        
-                        
+                        <msItem xml:id="ms_i1">
+                          <locus from="4r"></locus>
+                          <title type="complete" ref="LIT1948Mashaf"></title>
+                          <textLang mainLang="gez"></textLang>
+                          <note>With many marginal notes, some of them in Amharic.</note>
```

Comment:
Maybe you could refer to the addition and state that the catalogue is not clear on whether these are later additions